Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -108,26 +109,20 @@ private CardImageUrls innerGenerateURL(CardDownloadData card, boolean isToken) {
         // basic cards by api call (redirect to img link)
         // example: https://api.scryfall.com/cards/xln/121/en?format=image
         if (baseUrl == null) {
+            String cn= card.getCollectorId() ;
+            if( cn.endsWith( "*" )) cn= cn.substring(0 , cn.length() -1) +"★" ;
+            else if( cn.endsWith( "+" )) cn= cn.substring(0 , cn.length() -1) +"†" ;
+            else if( cn.endsWith( "Ph" )) cn= cn.substring(0 , cn.length() -2) +"Φ" ;
             baseUrl = String.format("https://api.scryfall.com/cards/%s/%s/%s?format=image",
                     formatSetName(card.getSet(), isToken),
-                    card.getCollectorId(),
+                    cn,
                     localizedCode);
-            alternativeUrl = String.format("https://api.scryfall.com/cards/%s/%s/%s?format=image",
+            alternativeUrl = String.format("https://api.scryfall.com/cards/%s/%s?format=image",
                     formatSetName(card.getSet(), isToken),
-                    card.getCollectorId(),
-                    defaultCode);
-            // workaround to use cards without english images (some promos or special cards)
-            // bug: https://github.com/magefree/mage/issues/6829
-            // example: Mysterious Egg from IKO https://api.scryfall.com/cards/iko/385/?format=image
-            if (Objects.equals(baseUrl, alternativeUrl)) {
-                // without loc code scryfall must return first available image
```

Comment:
Must keep original comments and examples about cards without loc and with non english cards.

Reply:
Returned original workaround comment, also added note about defaulting to an english image when one is available.